src/skycoin/config.go

```diff
@@ -587,6 +591,7 @@ func (c *NodeConfig) RegisterFlags() {
 	flag.BoolVar(&c.EnableGUI, "enable-gui", c.EnableGUI, "Enable GUI")
 	flag.BoolVar(&c.EnableUnversionedAPI, "enable-unversioned-api", c.EnableUnversionedAPI, "Enable the deprecated unversioned API endpoints without /api/v1 prefix")
 	flag.BoolVar(&c.DisableCSRF, "disable-csrf", c.DisableCSRF, "disable CSRF check")
+	flag.BoolVar(&c.DisableHeadercheck, "disable-headercheck", c.DisableHeadercheck, "disable Header check")
```

Comment:
Needs a better description, "Header check" is ambiguous, it disabled the Host, Origin and Referer checks